eclipse_plugin/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target/
+/.mvn/wrapper/maven-wrapper.jar
```

Comment:
I think this file can be deleted now that we're using gradle

Reply:
I can take care of this, there are a few gradle-shaped things I plan to experiment with as well (I'd like to pull the list of dependencies listed in the manifest dynamically, and wire up plugins for maven publishing)

Reply:
Go ahead, my Gradle knowledge is quite limited, so I limited myself to a static `MANIFEST.MF`. However the `Bundle-Classpath` can be certainly generated dynamically, so that the deps keep their original names.

**Remark**: some dependencies like `guava` or `jackson` are already in the official Eclipse P2 repositories. I just don't know how to force Eclipse to install additional bundles, probably this would require an "Eclipse feature" distributed in a P2 repository.